lms/djangoapps/courseware/tests/test_footer.py

```diff
@@ -36,26 +37,26 @@ class TestFooter(TestCase):
         "youtube": "https://www.youtube.com/"
     }
 
+    @with_is_edx_domain(True)
```

Comment:
Would the tests specific to edx.org eventually be moved out of the main codebase? To the edx.org theme maybe?